use-cases/financial_payment_classification/financial payment classification.ipynb

```diff
@@ -0,0 +1,1849 @@
+{
```

Comment:
<p>Style nit: you could use a larger header for the notebook, such as <code># SageMaker Payment Classification</code> ﻿</p><p> </p><p>Grammar (missing word): <code>deploy a machine learning</code>  -&gt; <code>deploy a machine learning model</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3017/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
solved. fixed grammar and updated heading formats